src/server/enrollment-system.js

```diff
@@ -241,19 +241,19 @@ function veteranToSpouseInfo(veteran) {
  */
 function resourceToIncomeCollection(resource) {
   const incomeCollection = [];
-  if (resource.grossIncome) {
+  if (resource.grossIncome !== undefined) {
```

Comment:
Ugh, that's true. In any case, I think we should have a check that checks for null and undefined, to be safe.